content/information-distribution/article.md

```diff
@@ -30,4 +30,48 @@ If the original news item contains media, these media **should** be uploaded to
 the node using [Matrix's content repository
 module](https://matrix.org/docs/spec/client_server/r0.4.0.html#id112).
 
-2️⃣: rest of the page
+## Matrix event `network.informo.article`
+
+This message event is meant to be sent by a source Matrix user. If the article
+is signed, it **must** be wrapped into a [Signed Matrix
```

Comment:
This part should be left as is until #8 gets merged. Afterwards, two issues need to be addressed:

1. "If the article is signed" => articles are always signed
2. No mention who signs the event

Both can be solved by removing the paragraph, or rephrasing it, since, as it currently is phrased, #8 makes it redundant.

Reply:
IMO we should consider #8 merged as it will likely be merged before this PR (thus the link to the signing page)

Reply:
Sure, I was suggesting to do it that way (then merge master into this branch) so we can have a full picture of the page's final state.